### PR TITLE
Generate custom cloudify manager image documentation

### PR DESCRIPTION
**Pull request by yaelpeled**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +9,58 @@ Please see [Bootstrapping using Docker](http://getcloudify.org/guide/3.1/install
...
+```
+
+* Run the [build.sh](https://github.com/cloudify-cosmo/cloudify-packager/blob/master/docker/build.sh) 
+	script from the [docker folder](https://github.com/cloudify-cosmo/cloudify-packager/tree/master/docker):
+```
+  cd cloudify-packager/docker/
+  . build.sh
+```
+
+* Create a tar file:
```
Comment (adamlavie) [inline README.md]: Create a tar file from the generated image

File: README.md [hunk 2]
```diff
@@ -9,6 +9,58 @@ Please see [Bootstrapping using Docker](http://getcloudify.org/guide/3.1/install
...
+	
+	- For example:
+		
+		- Replace the master branch with a specific branch for one or more modules in [modules](https://github.com/cloudify-cosmo/cloudify-packager/blob/master/docker/vars.py#L123).
+		- Add packages to one of the requirement lists so it will be installed on the image.<br>
+		
+		  For example, add the package "my-package" to the manager's requirements list (the [reqs](https://github.com/cloudify-cosmo/cloudify-packager/blob/master/docker/vars.py#L119) list):
+			
+```
+		"managera": {
```
Comment (adamlavie) [inline README.md]: manager

File: README.md [hunk 3]
```diff
@@ -9,6 +9,58 @@ Please see [Bootstrapping using Docker](http://getcloudify.org/guide/3.1/install
...
+### Generate a custom Cloudify manager image
+
+* Clone the cloudify-packager repository from github:<br>
+`git clone https://github.com/cloudify-cosmo/cloudify-packager.git`
+
+* Make your changes in [var.py](https://github.com/cloudify-cosmo/cloudify-packager/blob/master/docker/vars.py)
+	
+	- For example:
+		
+		- Replace the master branch with a specific branch for one or more modules in [modules](https://github.com/cloudify-cosmo/cloudify-packager/blob/master/docker/vars.py#L123).
```
Comment (adamlavie) [inline README.md]: Cloudify related dependencies

**Comment by adamlavie** [comment]
I was really going to merge it yesterday but travis was stuck until late yesterday.. ;)

________________________________

**Comment by yaelpeled** [comment]
I know , I didn't want to bother you at night ;)

________________________________

**Pull request merged**